app/models/steps/individual.rb

```diff
@@ -54,7 +54,7 @@ def restart
     end
 
     def user_is_not_requester
-      if user_id && user_id == proposal.requester_id
+      if user && user == proposal.requester
```

Comment:
my guess is that the use of `user_id` instead of `user` and `requester_id` instead of `requester` is to minimize the amount of SQL statements generated. By using the AR objects instead, it adds 2 more statements. 

Is that a misguided, false laziness (preferring present integers over possibly-not-yet-loaded AR objects)?